src/hooks/use-connect-editor.ts

```diff
@@ -66,6 +70,14 @@ export const useConnectEditor = () => {
     storeDispatch({ hasPendingSuggestions })
   }, [storeDispatch, hasPendingSuggestions])
 
+  const warningsCount = useMemo(() => {
+    return detectInconsistencyPluginKey.getState(state)?.warnings?.length || 0
+  }, [state])
+
+  useEffect(() => {
+    storeDispatch({ warningsCount })
```

Comment:
would that be worth just putting all the warning in the store instead of just count? they are going to be used the inspector anyway. What do you think?